pipelines/optimus/adapter.wdl

```diff
@@ -140,7 +141,7 @@ workflow AdapterOptimus {
   Boolean record_http = false
   Boolean add_md5s = false
 
-  String pipeline_tools_version = "v0.57.0"
+  String pipeline_tools_version = "se-add-optimus-chemistry-param"
```

Comment:
You'll update this to a versioned release, right? :)

Reply:
Yes! This is just for testing :)

Reply:
Done making that change @rhiananthony!